src/libtorch.cc

```diff
@@ -1532,9 +1528,7 @@ SetStringInputTensor(
                   std::to_string(element_idx + 1) + " for inference input '" +
                   name + "', expecting " + std::to_string(request_element_cnt))
                   .c_str()));
-      FillStringTensor(
-          input_list, tensor_offset + element_idx,
-          request_element_cnt - element_idx);
+      FillStringTensor(input_list, request_element_cnt - element_idx);
```

Comment:
I don't think you need to fill the rest of string elements in this case, when `element_idx >= request_element_cnt` evaluates to true, there is `element_idx+1` elements in `input_list`

Reply:
Thanks for the catch. Removed this line.